lib/themes/dosomething/paraneue_dosomething/scss/content/_onboarding.scss

```diff
@@ -162,7 +162,7 @@
 
 .slideshow__controller {
   display: table;
-  margin: 0 auto;
+  margin: $base-spacing auto;
```

Comment:
You should probs be using padding for this instead of margins, since sibling element's margins collapse, which can cause unwanted visual discrepancies. Consider updating the `padding` property below.